<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProperty</class>
 <widget class="QWidget" name="AddProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_11">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Property Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineProperty"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineDesc"/>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>WidgetType:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioLine">
         <property name="text">
          <string>LineEdit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="radioRadio">
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="radioCheck">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioSpin">
         <property name="text">
          <string>SpinBox</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="radioEnableWidget">
         <property name="text">
          <string>Enable Widget</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="radioUn">
         <property name="text">
          <string>unknown</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupLine">
     <property name="title">
      <string>LineEdit</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>DataType:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboLineEdit">
          <item>
           <property name="text">
            <string>int</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>text</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>float</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>file</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Default Value:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupCheck">
     <property name="title">
      <string>CheckBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Default Value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioCheckChecked">
          <property name="text">
           <string>Checked</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioCheckUnchecked">
          <property name="text">
           <string>Unchecked</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupSpin">
     <property name="title">
      <string>SpinBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Default Value:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinDefault"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Min Value:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinMin"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Max Value:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinMax"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QGroupBox" name="groupRadio">
     <property name="title">
      <string>RadioButton</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" rowspan="2">
         <widget class="QListWidget" name="listRadio">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="btnAddRadio">
          <property name="text">
           <string>Add Selection</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="btnDelRadio">
          <property name="text">
           <string>Del Selection</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QGroupBox" name="groupHide">
     <property name="title">
      <string>EnableWidget</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>DataType:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboEnableWidget">
          <item>
           <property name="text">
            <string>int</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>text</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>float</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>file</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Default Value:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEnableWidget"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Enabled:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkEnableWidget">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioLine</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupLine</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupRadio</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupCheck</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioEnableWidget</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupHide</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>639</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioSpin</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupSpin</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>AddProperty</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>805</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>800</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
